### PR TITLE
Repair similarity index after group unmerge.

### PR DESCRIPTION
**Pull request by tkaemming**

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: tests/sentry/tasks/test_unmerge.py [hunk 1]
```diff
@@ -273,6 +276,10 @@ def create_message_event(template, parameters):
             (u'sentry:release', u'version', 17),
         ])
 
+        assert features.query(source) == [
+            (source.id, {'message:message:character-shingles': 1.0}),
+        ]
```
Comment (tkaemming) [inline tests/sentry/tasks/test_unmerge.py]: This is super brittle with regard to changes to feature extraction, I can't think of a better way to do this off of my head that doesn't involve dependency injection and I really don't want to open that can of worms right now if possible.

Comment (JTCunning) [inline tests/sentry/tasks/test_unmerge.py]: Eh, think it's fine. It'll also just be a quick change whenever extraction is changed.

**Review by JTCunning — APPROVED** [review]
Good stuff.

**Pull request merged**